src/Native/System.IO.Compression.Native/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+project(System.IO.Compression.Native)
```

Comment:
Yeah, my plan was to modify the CMakeLists to do different things depending on the Target OS - Unix would execute Eric's code, Windows would execute mine. 

The shared code between the two is narrow enough that separate CMakeLists may be better as you said.